jalib/jalloc.h

```diff
@@ -143,8 +144,9 @@ class JAlloc
       return JAllocDispatcher::free(p);
     }
 
```

Comment:
As earlier, I think these changes are no longer needed.  Possibly you can delete this entire commit.